app/assets/stylesheets/style.scss

```diff
@@ -14,6 +14,13 @@ body {
 label {
   color: white;
 }
+a {
+  color: black;
+    background:rgba(31,138,150,0.95);
+}
+a:hover{
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.